var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -41,6 +41,9 @@ class Ncurses(AutotoolsPackage):
     variant('symlinks', default=False,
             description='Enables symlinks. Needed on AFS filesystem.')
 
+    variant('wide', default=True,
+            description='Enable to build ncuresesw, disable to build ncurses')
```

Comment:
One would need to dig into this, but the may still have same symbols and therefore could be conflicting within a single DAG